vendor/bat-native-ledger/src/bat/ledger/internal/media/twitter.cc

```diff
@@ -132,46 +134,34 @@ bool Twitter::IsExcludedPath(const std::string& path) {
     return true;
   }
 
-  const std::vector<std::string> paths({
-      "/",
-      "/settings",
-      "/explore",
-      "/notifications",
-      "/messages",
-      "/logout",
-      "/search",
-      "/about",
-      "/tos",
-      "/privacy",
-      "/home"
+  const std::vector<std::string> screen_names({
```

Comment:
how will this change affect android and ios, where they don't have content scripts and just relay on url's

Reply:
Yep, good point. I think the best thing to do in that case is to test the query-string first and if not found there, test it as we did in the past. I'll update to address that.

Reply:
Fixed, we now try both. Tested on Windows and Android.